spec/controllers/api/v1/channels_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ChannelsController, type: :controller do
+  let!(:organization) { create(:organization) }
+  let!(:channel) { create(:channel, organization: organization) }
+
+  describe 'GET #show' do
+    it 'return channel' do
+      request.headers['token'] = organization.token
+      get :show, params: { id: channel.id }
+      json_response = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      expect(json_response['id']).to eq(channel.id)
+    end
+
+    it 'return error by invalid token' do
+      request.headers['token'] = organization.token + 'Ruby'
+      get :show, params: { id: channel.id }
+      expect(response).to have_http_status(400)
+    end
+
+    it 'return error by non exit channel' do
+      request.headers['token'] = organization.token
+      get :show, params: { id: 44 }
+      expect(response).to have_http_status(400)
```

Comment:
small proposal, if u prefer to use block-styled tests, could you pls at least separate logical units with empty lines or comments, so that code wouldn't look like a text file

just to compare
```
    it 'return error by non exit channel' do
      request.headers['token'] = organization.token
      get :show, params: { id: 44 }
      expect(response).to have_http_status(400)
  end
```

```
it 'returns error on non-existing channel' do
  request.headers['token'] = organization.token

  get :show, params: { id: 44 }

  expect(response).to have_http_status(400)
end
```